src/app/shared/layout/header.component.ts

```diff
@@ -31,7 +31,7 @@ export class HeaderComponent implements OnInit {
               public auth: AuthService,
               public phaseService: PhaseService,
               public userService: UserService,
-              private loggingService: LoggingService,
+              public loggingService: LoggingService,
```

Comment:
This change overlaps #730. Depending on how #730 is resolved, it would be good to check and make sure the merge conflict here is correct.
Just a note